vars/fcosKola.groovy

```diff
@@ -58,6 +59,16 @@ def call(params = [:]) {
             shwrap("test -d ${cosaDir}/tmp/kola-upgrade")
         }
     }
+    if (params["selfTest"]) {
+        kolaRuns['self_tests'] = {
+            try {
+                shwrap("cd ${cosaDir} && ${env.WORKSPACE}/ci/run-kola-self-tests")
```

Comment:
Sorry, I missed this before. I don't think we should rely on this script here. If we want to support the self-test as target in `fcoSKola`, we shouldn't assume that it's going to be running against a cosa git checkout. (E.g. another place where I see this being useful is in the https://github.com/coreos/fedora-coreos-config CI since it has the code that gathers the journal logs).

So here I think we should call out to `kola` ourselves like that script does.